tests/sentry/issues/test_utils.py

```diff
@@ -49,6 +51,7 @@ def build_occurrence_data(self, **overrides: Any) -> IssueOccurrenceData:
             "type": ProfileFileIOGroupType.type_id,
             "detection_time": datetime.now().timestamp(),
             "level": "warning",
+            "initial_issue_priority": PriorityLevel.LOW,
```

Comment:
Is it better to not include this here and rely on the default being set correctly? Otherwise we might break existing external customers

Reply:
In that case, should I remove the explicit priority i've set for files like these? With the defaults, this shouldn't be necessary but I'm not sure if it's better to be explicit 
 
crons: `src/sentry/monitors/logic/mark_failed.py` 
replays: `src/sentry/monitors/logic/mark_failed.py`
statistical detectors: `src/sentry/monitors/logic/mark_failed.py`

Reply:
Yeah, I was thinking about that too. I think that it's probably unnecessary, since we shouldn't really need the customer to pass the default for us, since we already know it. Passing priority is mostly needed for when/if a customer wants to send us different priorities based on some internal logic specific to their product.

Reply:
makes sense, I'll clean that up!